support/ebpf/extmaps.h

```diff
@@ -18,6 +18,7 @@ extern bpf_map_def inhibit_events;
 extern bpf_map_def interpreter_offsets;
 extern bpf_map_def system_config;
 extern bpf_map_def trace_events;
+extern bpf_map_def go_procs;
```

Comment:
```suggestion
extern bpf_map_def golabel_procs;
```
I think it is not currently easy to share per-PID maps between different interpreter types, so this probably should be specific to `golabel`.

Reply:
I haven't looked at sharing them but we're already sharing the golabels procs between the interpreter program and the go_labels program.   Probably best to keep them separate so they can each be turned on/off independently.